services/minespace-web/src/components/Forms/projectSummaries/AddEditProjectSummaryForm.js

```diff
@@ -49,13 +49,6 @@ export class AddEditProjectSummaryForm extends Component {
   render() {
     return (
       <Form layout="vertical" onSubmit={this.props.handleSubmit}>
-        <Field
-          id="project_summary_date"
-          name="project_summary_date"
-          label="Date"
-          placeholder="Please select date"
-          component={renderConfig.DATE}
-        />
         <Field
```

Comment:
I thought we weren't going to remove this field in this ticket?

Reply:
Hmm, I think perhaps I got mixed up with what the end result was supposed to be. I can easily revert this if its a problem.

Reply:
yeah lets either: 
1. add it back
2. remove it, but populate the received date on the BE so we can still submit project summaries. 

I vote for 1

Reply:
Alright, I will bring it back in, thanks for the feedback